src/stories/form.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
@matiasbavera what is your take on this?

Reply:
One of the multiple objectives of the Storybook is to use it like "style guide". If someone wants to contribute to the project, it can see how we design our components (in this case which components we use and how `Material-ui `designed its components).

I think it will not hurt to have this here and in case we want to eventually put our own touch of style on `Material-ui `components this is going to be a good place to demonstrate it.

You can check storybook examples here https://storybook.js.org/docs/examples/

If none of the reasons convinces you, you can remove it without hesitation =)

Reply:
Make sense. I will leave it here for future changes.